Stratis.Bitcoin/Features/BlockStore/BlockRepository.cs

```diff
@@ -12,32 +12,33 @@ namespace Stratis.Bitcoin.Features.BlockStore
     public interface IBlockRepository : IDisposable
     {
         Task Initialize();
-
         Task PutAsync(uint256 nextBlockHash, List<Block> blocks);
-
         Task<Block> GetAsync(uint256 hash);
-
         Task<Transaction> GetTrxAsync(uint256 trxid);
-
         Task DeleteAsync(uint256 newlockHash, List<uint256> hashes);
-
         Task<bool> ExistAsync(uint256 hash);
 
         Task<uint256> GetTrxBlockIdAsync(uint256 trxid);
-
         Task SetBlockHash(uint256 nextBlockHash);
-
         Task SetTxIndex(bool txIndex);
+
+        uint256 BlockHash { get; }
+        BlockStoreRepositoryPerformanceCounter PerformanceCounter { get; }
+        bool TxIndex { get; }
     }
 
     public class BlockRepository : IBlockRepository
     {
         protected readonly DBreezeSingleThreadSession session;
         protected readonly Network network;
+
+        protected static readonly byte[] blockHashKey = new byte[0];
         protected HashSet<string> tableNames = new HashSet<string>() { "Block", "Transaction", "Common" };
-        protected static readonly byte[] BlockHashKey = new byte[0];
-        protected static readonly byte[] TxIndexKey = new byte[1];
+        protected static readonly byte[] txIndexKey = new byte[1];
```

Comment:
Why make (txIndexKey ,blockHashKey ) lower case? it might effect IndexStore (FYI)